tests/kernel/interrupt/src/interrupt_util.h

```diff
@@ -55,19 +60,8 @@ static void trigger_irq(int irq)
 #endif
 }
 
-#elif defined(CONFIG_RISCV)
```

Comment:
@stephanosio can you make a note that for RISCv the test does not run any more and link to the respective github issue?

Reply:
There is an issue about this:
https://github.com/zephyrproject-rtos/zephyr/issues/23593

As far as I can see, since this test should have never been enabled for RISC-V in the first place, I think this is really a non-issue.

https://github.com/zephyrproject-rtos/zephyr/issues/23593#issuecomment-605370756